doc-content/enterprise-only/project-data/reviewing-change-requests-proc.adoc

```diff
@@ -0,0 +1,24 @@
+[id='reviewing-change-requests-proc_{context}']
+
+= Working with change requests
+You can view change requests for any branch that you have access to. You must have administrator permissions to accept a change request.
+
+.Prerequisites
+* You have more that one branch of a {CENTRAL} project.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and select a space and project.
+. On the project page, verify that you are on the correct branch.
++
+image::getting-started/change-branch.png[Selecting a branch menu]
+. Click the *Change Requests* tab. A list of pending change requests appears.
+. To filter change requests, select *Open*, *Closed*, or *All* to the left of the *Search* box.
+. To search for specific change requests, enter an ID or text in the *Search* box and click the magnifying glass.
+. To view the change request details, click the summary link. The change request window has two tabs:
+.. Review the *Overview* tab for general information about the change request.
+.. Click the *Changed Files* tab and expand a file to review the proposed changes.
+. Click a button in the top right corner.
+.. Click *Accept* to merge the changes into the target branch.
+.. Click *Reject* to reject the changes and leave the target branch unchanged.
+.. Click *Close* to close the change request without rejecting or accepting it.
```

Comment:
Perhaps we can highlight here that the _Close_ operation is only available to the user who has submitted the change request, not for everyone.

Reply:
Updated.